test/preview.ts

```diff
@@ -1,81 +1,136 @@
-import path from 'path'
-import { File } from '../src/file'
-import * as preview from '../src/preview'
+import { enterTestMode } from 'carlo'
+import { File, FileType } from '../src/file'
+import { Preview, fileToURI } from '../src/preview'
+import { CLIError } from '../src/error'
+
+beforeAll(() => enterTestMode())
+
+describe('Preview', () => {
+  const previews = new Set<Preview>()
+  const preview = (opts = {}) => {
+    const instance = new Preview(opts)
+    previews.add(instance)
+
+    return instance
+  }
+
+  afterEach(async () => {
+    for (const instance of previews)
+      if (instance.carlo)
+        await instance.carlo
+          .browserForTest()
+          .process()
+          .kill()
```

Comment:
First I tried `await instance.carlo.browserForTest().close()`, but sometimes it throws `read ECONNRESET` in remaining tests.